lms/templates/courseware/info.html

```diff
@@ -59,37 +50,42 @@ <h2 class="page-subtitle">${course.display_name | h}</h2>
     % endif
   </div>
   <div class="info-wrapper">
-    % if user.is_authenticated():
-    <section class="updates">
-      % if studio_url is not None and masquerade and masquerade.role == 'staff':
-      <div class="wrap-instructor-info studio-view">
-        <a class="instructor-info-action" href="${studio_url}">
-          ${_("View Updates in Studio")}
-        </a>
-      </div>
+    <div class="course-updates">
+      % if user.is_authenticated():
+        <section class="updates">
+          % if studio_url is not None and masquerade and masquerade.role == 'staff':
+            <div class="wrap-instructor-info studio-view">
+              <a class="instructor-info-action" href="${studio_url}">
+                ${_("View Updates in Studio")}
+              </a>
+            </div>
+          % endif
+          <h1 class="course-info-heading">${_("Course Updates and News")}</h1>
+          ${get_course_info_section(request, course, 'updates')}
+        </section>
+        % if SelfPacedConfiguration.current().enable_course_home_improvements:
+          <section aria-label="${_('Important Course Dates')}" class="course-dates">
+            <h1 class="course-info-heading">${_("Important Course Dates")}</h1>
+            ${get_course_date_summary(course, user)}
+          </section>
+        % endif
+        % else:
+        <section class="updates">
+          <h1 class="course-info-heading">${_("Course Updates &amp; News")}</h1>
```

Comment:
"Course Updates &amp; News" here, but ""Course Updates and News" above?  Can we pick one and be consistent (also helps with translations)